internal/service/privatelinkendpointservicedatafederationonlinearchive/resource_privatelink_endpoint_service_data_federation_online_archive_test.go

```diff
@@ -13,19 +13,19 @@ import (
 )
 
 var (
-	resourceName            = "mongodbatlas_privatelink_endpoint_service_data_federation_online_archive.test"
-	projectID               = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
-	endpointID              = os.Getenv("MONGODB_ATLAS_PRIVATE_ENDPOINT_ID")
-	customerEndpointDNSName = os.Getenv("MONGODB_ATLAS_PRIVATE_ENDPOINT_DNS_NAME")
-	comment                 = "Terraform Acceptance Test"
-	atlasRegion             = "US_EAST_1"
+	resourceName = "mongodbatlas_privatelink_endpoint_service_data_federation_online_archive.test"
+	comment      = "Terraform Acceptance Test"
+	atlasRegion  = "US_EAST_1"
 )
 
 func TestAccNetworkPrivatelinkEndpointServiceDataFederationOnlineArchive_basic(t *testing.T) {
-	// Skip because private endpoints are deleted daily from dev environment
-	acc.SkipTestForCI(t)
+	var (
+		projectID  = acc.ProjectIDGlobal(t)
+		endpointID = os.Getenv("MONGODB_ATLAS_PRIVATE_ENDPOINT_ID")
```

Comment:
As this is used in every test now. Does it make sense to put it into a function in `acc` package?

Reply:
there is a precheck function: PreCheckPrivateEndpoint, do you mean that?   
or do you mean to get the value from the env.var? (we're getting env.vars directly from all tests)

Reply:
I meant getting the env-var directly from the tests.
Yes, I guess that is ok since we are doing that for all tests 👍